examples/asrc_demo/src/ff_appconf.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 XMOS LIMITED.
```

Comment:
It's a new file but date doesn't throw an error on source check?

Reply:
I don't think there any source check being done in sln_voice Jenkins. I'll fix the copyrights on all files.